openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -97,6 +98,7 @@ class Meta(object):
     course_video_url = TextField(null=True)
     effort = TextField(null=True)
     self_paced = BooleanField(default=False)
+    instructor_info = JSONField(null=False, blank=False)
```

Comment:
Ah, got it. Yes, I had assumed that the structure of the data was a list of dicts. Let's not change the structure of the data at this point. So the default should be a dict and not a list.